demisto_sdk/commands/format/update_generic_json.py

```diff
@@ -67,8 +67,9 @@ def remove_null_fields(self):
             a = yaml.safe_load(file_obj)
```

Comment:
```suggestion
            schema_data = yaml.safe_load(file_obj)
```


A good opportunity to change a bad name to a more useful one